dbschema/producible.esdl

```diff
@@ -4,7 +4,20 @@ module default {
       delegated constraint exclusive;
     };
     
-    scriptureReferences: multirange<int32>;
+    scripture: Scripture::Collection {
+      on source delete delete target if orphan;
+      # https://github.com/edgedb/edgedb/issues/5827
+      # rewrite insert, update using (
+      #   if exists .scripture.verses then .scripture else <Scripture::Collection>{}
+      # );
```

Comment:
This would be my ideal, but since it's bugged I used the trigger below to at least error on empty scripture collections.

DerivativeScriptureProducts have the option to override the scripture defined on their producible.
I wanted to leave scripture collections empty-able, at least for now, to leave the option to support an empty collection serving as an override meaning the product produces no scripture, even though it's attached to a producible that does.

There's probably a cleaner way, but I think products will be re-thought shortly after migration, so I'm trying not to change too much right now.